From a562fd516a7970e80d50cc163d78c07a425ba441 Mon Sep 17 00:00:00 2001
From: Elvis Dukaj <elvis.dukaj@gmail.com>
Date: Wed, 22 Jan 2025 11:36:21 +0100
Subject: [PATCH 1/2] fix: conan requirements

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 93c9618..17fb937 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.13)
 
 project(dvm LANGUAGES C)
 
+find_package(libogc CONFIG REQUIRED)
+
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 	if(NINTENDO_DS AND ARM9)
 		set(CMAKE_INSTALL_PREFIX "${NDS_ROOT}" CACHE PATH "" FORCE)
@@ -80,6 +82,8 @@ if(NINTENDO_GAMECUBE OR NINTENDO_WII)
 	target_sources(dvm PRIVATE source/dvm_libogc.c)
 endif()
 
+target_link_libraries(dvm PUBLIC libogc::libogc)
+
 include(GNUInstallDirs)
 
 # Install the library
-- 
2.47.1

